release/models/system/openconfig-system-controlplane.yang

```diff
@@ -199,6 +214,186 @@ module openconfig-system-controlplane {
+              container management {
+                description
+                  "Traffic targeted this device used management - ssh,
+                  telnet, ftp, gN*, snmp, netconf, rest, etc";
+                container state {
+                  description
+                    "control-plane protection management traffic counters";
+                  config false;
+                  uses system-controlplane-built-in-limits-traffic-state;
+                }
+              }
+              container remaining {
+                description "all other packets send to control plane";
+                container state {
+                  description
+                    "control-plane protection remaining packet counters";
+                  config false;
+                  uses system-controlplane-built-in-limits-traffic-state;
+                }
+              }
+            }
+            container vendor {
+              description
+                "An anchor point for vendor and platform specific
+                extensions.
+                As each platform  and vendor has different implementation and internal
+                parts where packets may be dropped at any point in time. Providing
+                vendor-specific counters provides better visibility into control-plane
+                traffic.
+                The recommended usage of this container is to create an augment at
+                /system/control-plane-traffice/ingress/built-in-limits/vendor that contains additional
+                vendor/platform specific containers.
+                e.g.
+                augment /system/control-plane-traffice/ingress/built-in-limits/vendor {
+                  container <vendor name> {
+                    container <platform name> {
+                      container <traffic-group-name>
+                        container config {
+                          leaf packet-max-rate {
+                              ...
+                           }
+                          leaf packet-burst-size {
+                             ...
+                          }
+                          ...
+                          }
+                          container state {
+                            leaf packet-max-rate {
+                              ...
+                            }
+                            leaf packet-burst-size {
+                              ...
+                            }
+                            ...
+                            leaf aggregate-counter-name {
+                              type string;
+                              description
+                                \"One of abstract counters defined under ../abstaract-counters/,
+                                such 'bgp' or 'L2control'.
+                                This self-documentation allows client to assotiate multiple vendor
+                                counters with vendor-neutral, abstract coarse counter\";
```

Comment:
This is a good idea.  I think this could be a leaf-ref (instead of a string) to the specific node referred to:


```suggestion
                                "One of built-in counters defined under ../built-in-limits/state/counters/,
                                such 'bgp' or 'l2-control'.
                                This self-documentation allows client to asssociate multiple vendor
                                counters with vendor-neutral, abstract, aggregated counters.";
```